zeppelin-zengine/src/test/java/org/apache/zeppelin/search/LuceneSearchTest.java

```diff
@@ -173,12 +174,12 @@ public void canIndexAndReIndex() throws IOException, InterruptedException {
     assertThat(results).isNotEmpty();
   }
 
-  @Test
+  //@Test
```

Comment:
A null check in `deleteNoteIndex` should be sufficient to re-enable this test.

Reply:
Fixed